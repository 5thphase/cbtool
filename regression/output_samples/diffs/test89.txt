EOF airunstate firsdetach rolelist vmcattach vmrepair 
aialter aisave firslist roleshow vmccleanup vmresize 
aiattach aishow firsshow shell vmcdetach vmrestore 
aicapture aisuspend hostalter statealter vmclist vmresume 
aidetach cldalter hostfail stateshow vmcrsalter vmrun 
aidrsalter cldattach hostlist stats vmcrsattach vmrunstate
aidrsattach clddefault hostrepair trace vmcrsdetach vmsave 
aidrsdetach clddetach hostshow typealter vmcrslist vmshow 
aidrslist cldlist monextract typelist vmcrsshow vmsuspend 
aidrsshow cldshow monlist typeshow vmcshow waitfor 
aifail echo msgpub unregister vmdetach waiton 
ailist expid patternalter viewlist vmdisplay waituntil 
airepair faultalter patternlist viewshow vmfail 
airesize faultlist patternshow vmalter vmlist 
airestore faultshow poollist vmattach vmlogin 
airesume firsalter register vmcalter vmmigrate 
airun firsattach rolealter vmcapture vmprotect 
