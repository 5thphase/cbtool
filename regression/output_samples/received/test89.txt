
Documented commands (type help <topic>):
========================================
EOF          airun       firsalter     register    vmattach     vmlist    
aialter      airunstate  firsattach    rolealter   vmcalter     vmlogin   
aiattach     aisave      firsdetach    rolelist    vmcapture    vmmigrate 
aicapture    aishow      firslist      roleshow    vmcattach    vmprotect 
aidetach     aisuspend   firsshow      shell       vmccleanup   vmrepair  
aidrsalter   cldalter    hostfail      statealter  vmcdetach    vmresize  
aidrsattach  cldattach   hostlist      stateshow   vmclist      vmrestore 
aidrsdetach  clddefault  hostrepair    stats       vmcrsalter   vmresume  
aidrslist    clddetach   hostshow      trace       vmcrsattach  vmrun     
aidrsshow    cldlist     monextract    typealter   vmcrsdetach  vmrunstate
aifail       cldshow     monlist       typelist    vmcrslist    vmsave    
ailist       echo        msgpub        typeshow    vmcrsshow    vmshow    
airepair     expid       patternalter  unregister  vmcshow      vmsuspend 
airesize     faultalter  patternlist   viewlist    vmdetach     waitfor   
airestore    faultlist   patternshow   viewshow    vmdisplay    waiton    
airesume     faultshow   poollist      vmalter     vmfail       waituntil 

Undocumented commands:
======================
exit  help  pause  quit

