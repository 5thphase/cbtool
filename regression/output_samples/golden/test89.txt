
Documented commands (type help <topic>):
========================================
EOF          aisave      firsalter     poollist    vmattach     vmmigrate 
aialter      aishow      firsattach    register    vmcalter     vmprotect 
aiattach     aisuspend   firsdetach    reset       vmcapture    vmrepair  
aicapture    appdev      firslist      rolealter   vmcattach    vmresize  
aidetach     appload     firsshow      rolelist    vmccleanup   vmrestore 
aidrsalter   appnoload   getrandnr     roleshow    vmcdetach    vmresume  
aidrsattach  appundev    hostalter     shell       vmclist      vmrun     
aidrsdetach  cldalter    hostfail      statealter  vmcrsalter   vmrunstate
aidrslist    cldattach   hostlist      stateshow   vmcrsattach  vmsave    
aidrsshow    clddefault  hostrepair    stats       vmcrsdetach  vmshow    
aifail       clddetach   hostshow      trace       vmcrslist    vmsuspend 
ailist       cldlist     monextract    typealter   vmcrsshow    waitfor   
airepair     cldshow     monlist       typelist    vmcshow      waiton    
airesize     echo        monpurge      typeshow    vmdetach     waituntil 
airestore    expid       msgpub        unregister  vmdisplay  
airesume     faultalter  patternalter  viewlist    vmfail     
airun        faultlist   patternlist   viewshow    vmlist     
airunstate   faultshow   patternshow   vmalter     vmlogin    

Undocumented commands:
======================
exit  help  pause  quit

